docs/content/docs/connectors/datastream/filesystem.md

```diff
@@ -797,3 +797,322 @@ before the job is restarted. This will result in your job not being able to rest
 pending part-files are no longer there and Flink will fail with an exception as it tries to fetch them and fails.
 
 {{< top >}}
+
+# File Source
```

Comment:
I would put the content of the File Source at the top, so above the File Sink content